src/components/TextInput/styles.tsx

```diff
@@ -1,15 +1,15 @@
 import styled from 'styled-components';
-import { Inter } from 'next/font/google';
-import COLORS from '../../styles/colors';
+import { Open_Sans as OpenSans } from 'next/font/google';
+import COLORS from '@/styles/colors';
 
-const inter = Inter({ subsets: ['latin'] });
+const openSans = OpenSans({ subsets: ['latin'] });
 
 export const InputText = styled.input<{ $error: boolean }>`
+  // is possible to inherit text styles from H4?
+  ${openSans.style}
   color: ${COLORS.existingInputColor};
   font-size: 1rem; // 16px
-  ${inter.style}
-  font-weight: 400;
-  line-height: 19.2px;
+  font-weight: 600;
   word-wrap: break-word;
   padding: 0.625rem 1.25rem; // 10px 20px 10px 20px;
   border-radius: 5px;
```

Comment:
noticed that the border weight here is 1 px on figma when i inspected with dev move (can check w/ david what he thinks looks appropriate). didn't check other specific values, but could be good to do a quick check that the styles match figma

Reply:
gotcha! yeah david said he would change border thickness to 2px on figma later. but the border-radius refers to radius of the curvature of the border! thanks for the reminder!